ovos_utils/log.py

```diff
@@ -77,18 +91,18 @@ def __init__(cls, name='OVOS'):
 
     @classmethod
     def init(cls, config=None):
-
+        from ovos_utils.xdg_utils import xdg_state_home
+        xdg_env = os.environ.get("OVOS_CONFIG_BASE_FOLDER")
         try:
             from ovos_config.meta import get_xdg_base
-            default_base = get_xdg_base()
+            xdg_base = xdg_env or get_xdg_base()
```

Comment:
should be the other way around, get_xdg_base already reads the env var internally

Reply:
dont see why the additional env var option should be made since it is already dine in `get_xdg_base`

what is done:

_when config package is present (which includes a env var check):_
`xdg_base = get_xdg_base()`
_otherwise:_
`xdg_base = os.environ.get("OVOS_CONFIG_BASE_FOLDER") or "mycroft"`

xdg_path = os.path.join(xdg_state_home(), xdg_base)